R/initModel.R

```diff
@@ -132,6 +134,11 @@ initModel <- function(config = NULL,
     copyGamsFiles(path, overwrite = !isFALSE(restart))
   }
 
+  # Copy the initial gdx. In restart runs, this can be the output gdx of the restarted folder
+  if ("useAsStart" %in% restart && file.exists(file.path(path, "output.gdx"))) {
+    cfg[["startingPoint"]] <- file.path(path, "output.gdx")
+    message("Using the output-gdx of the restarted run as starting point.")
+  }
```

Comment:
I only just noticed that I could have used just ```file.path(path)```, because the file name is added  in ```copyInitialGdx```. If you have any preferences, feel free to let me know :-)

Reply:
I like it the way you implemented it because it is more specific then letting `copyInitialGdx` decide which file to consider.